### ISSUE TITLE
metadata.min.json is always bundled even when using custom metadata?

### ISSUE THREAD

**Issue by msheakoski** [catamphetamine/libphonenumber-js#254]
Following the instructions from 
https://github.com/catamphetamine/libphonenumber-js#customizing-metadata I generated a metadata.custom.json with only a few countries to make it smaller and using the same ES6 example code, I imported it with `import metadata from './metadata.custom.json'`.

The resulting webpack bundle does include my metadata.custom.json (3.5 KB), but also still includes metadata.min.json (88 KB).

How can I prevent this from happening?

<img width="589" alt="screen shot 2018-09-26 at 11 53 19 am" src="https://user-images.githubusercontent.com/4156/46092436-e4014d80-c182-11e8-8802-cabaded7fda9.png">

**Comment by catamphetamine** [comment]
There's no evidence that `metadata.min.json` is included when using `/custom` functions.
The only places where it's imported are `index.common.js` and `index.es6.js` so unless you're using those it won't bundle it.

**Comment by msheakoski** [comment]
@catamphetamine Thank you. Your reply helped me track down what was happening. I forgot that I was using the Metadata class directly to do some normalization before parsing. I used TypeScript's [module augmentation](https://www.typescriptlang.org/docs/handbook/declaration-merging.html#module-augmentation) feature to add the Metadata typings and picked the wrong module name:

```typescript
import {Metadata} from "libphonenumber-js"; // from index.es6.js
const metadata = new Metadata(require("./metadata.custom.json"));
```
I changed it to
```typescript
import Metadata from "libphonenumber-js/es6/metadata"; // from es6/metadata.js
```
and that solved the issue.
